### PR TITLE
[AutoPR] restapi_auto_datafactory/resource-manager

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

File: azure-mgmt-datafactory/azure/mgmt/datafactory/version.py [hunk 1]
```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.4.0"
+VERSION = "2017-09-01-preview"
```
Comment (hvermis) [inline azure-mgmt-datafactory/azure/mgmt/datafactory/version.py]: Is this supposed to be like this? The changes look good, and I will update the version later separately unless the scripts have changed and now we put API version here.

Comment (lmazuel) [reply azure-mgmt-datafactory/azure/mgmt/datafactory/version.py]: Nope, this has to be the package version. This info should be in your Readme.md file. If you have more Swagger PR to do, use one of them to update the Python conf to add `package-version: 0.4.1` (or whatever the good value). If you don't have, no big deal we will change this one just before merging.

Comment (hvermis) [reply azure-mgmt-datafactory/azure/mgmt/datafactory/version.py]: We might have on more, still waiting on that one.

**Comment by lmazuel** [comment]
@AutorestCI rebase

**Pull request merged**